### DISCUSSION TITLE
The more on the following page has disappeared

### DISCUSSION THREAD

**Discussion by RookieJunChen** [community/community#38068] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/37040228/199518257-08609ca8-6279-4118-a310-3a3acb8d83d3.png)
![image](https://user-images.githubusercontent.com/37040228/199518374-d34064dc-ca02-4e29-a62b-fec723ff1f91.png)
![image](https://user-images.githubusercontent.com/37040228/199519268-1879527a-a322-406f-92e6-90189c9324b3.png)
Previously, on the _Following_ page, there was a _More_ option to see more content, but it has recently disappeared. While _For you_ has the option of _More_.
I would like to ask you to encounter this situation? How to restore it to the way it was with _More_？

**Comment by david-engelmann** [comment]
You should see a previous and next section (see image below).

![image](https://user-images.githubusercontent.com/81196676/199520074-07209a39-d4ac-4969-9e75-9708cab4f133.png)

Are you on the desktop or mobile version of Github?

**Comment by RookieJunChen** [comment]
![image](https://user-images.githubusercontent.com/37040228/199634281-6dd379cd-a529-4565-938c-578312dede03.png)
What I mean by "Following" here is that when you click on the circle pattern of github, something like the dynamics of the person you are following comes out.
I am using the web page github here.

**Comment by RookieJunChen** [comment]
I don't know why it's back to normal, I don't need to seek help on this anymore, thanks!

**Comment by jfullstackdev** [comment]
it can happen when there are no more feeds to load sometimes then at other times, it will have `load more` because you have more feeds at that time
